kernel/packages/shared/loading/ReportFatalError.ts

```diff
@@ -43,22 +44,26 @@ export function BringDownClientAndShowError(event: ExecutionLifecycleEvent) {
     event === COMMS_COULD_NOT_BE_ESTABLISHED
       ? 'comms'
       : event === NOT_INVITED
-      ? 'notinvited'
-      : event === NO_WEBGL_COULD_BE_CREATED
-      ? 'notsupported'
-      : event === MOBILE_NOT_SUPPORTED
-      ? 'nomobile'
-      : event === NEW_LOGIN
-      ? 'newlogin'
-      : event === NETWORK_MISMATCH
-      ? 'networkmismatch'
-      : event === AVATAR_LOADING_ERROR
-      ? 'avatarerror'
-      : 'fatal'
+        ? 'notinvited'
+        : event === NO_WEBGL_COULD_BE_CREATED
+          ? 'notsupported'
+          : event === MOBILE_NOT_SUPPORTED
+            ? 'nomobile'
+            : event === NEW_LOGIN
+              ? 'newlogin'
+              : event === NETWORK_MISMATCH
+                ? 'networkmismatch'
+                : event === AVATAR_LOADING_ERROR
+                  ? 'avatarerror'
+                  : 'fatal'
 
   globalThis.globalStore && globalThis.globalStore.dispatch(fatalError(targetError))
   Html.showErrorModal(targetError)
   aborted = true
+
+  if (window.Rollbar) {
+    window.Rollbar.configure({ enabled: false })
```

Comment:
If we disable `Rollbar` immediately after logging the error, you are certain that the error web request that got sent earlier in `ReportFatalError` is not aborted?

Reply:
Let me check it...

Reply:
Checked and confirmed: the previous `ReportFatalError` is being sent correctly before deactivate Rollbar. For check it, I have done this:
1. I have forced to sent a `window.Rollbar.critical(new Error('TEST ERROR BEFORE DEACTIVATE ROLLBAR!!'), {})` just BEFORE the deactivation and a `window.Rollbar.critical(new Error('TEST ERROR AFTER DEACTIVATE ROLLBAR!!'), {})` just AFTER.

![image](https://user-images.githubusercontent.com/64659061/124787904-2b7ab200-df49-11eb-84fb-e580aa3f68e9.png)

2. I have forced to make a call `BringDownClientAndShowError(...)`.
3. Checking the Network section in the chrome developer tools, we can see that the request related to the first fatal error has been sent but not the second one.

![image](https://user-images.githubusercontent.com/64659061/124787592-de96db80-df48-11eb-9a34-8e2c0c3fb83d.png)

Reply:
Amazing!